content/spin/v2/observing-apps.md

```diff
@@ -0,0 +1,39 @@
+title = "Observing Applications"
+template = "spin_main"
+date = "2023-11-04T00:00:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/observing-apps.md"
+
+---
+
+- [Logs](#logs)
+- [OpenTelemetry (OTEL)](#opentelemetry-otel)
+
+## Logs
+
+Spin handles application logs by default, storing output and error messages from file system-run applications in the `.spin/logs` directory alongside the manifest file's location. Users have the option to direct logs to a specific folder using the `--log-dir` flag of the `spin up` command. Additionally, if users wish to prevent `stdout` and `stderr` from being written to disk, they can specify an empty string for the `--log-dir` flag, i.e. `spin up --log-dir ""` - effectively disabling log storage. See the [persistent logs](./running-apps#persistent-logs) section for more details.
+
+## OpenTelemetry (OTEL)
+
+Spin now has experimental support for the [OpenTelemetry (OTEL)](https://opentelemetry.io/) observability standard. When configured, Spin will now emit traces of your Spin App as an OTEL [signal](https://opentelemetry.io/docs/concepts/signals/). Here is a teaser of how to configure it.
```

Comment:
```suggestion
Spin has experimental support for the [OpenTelemetry (OTEL)](https://opentelemetry.io/) observability standard. When configured, Spin will emit traces of your Spin App as an OTEL [signal](https://opentelemetry.io/docs/concepts/signals/). Here is a teaser of how to configure it.

> OpenTelemetry support requires Spin 2.4 or above.
```

(reason: "now" is a point in time which works well in a blog post, but for persistent docs it's not clear what it means)